azure-pipelines.yml

```diff
@@ -54,522 +54,568 @@ resources:
       ref: refs/tags/v1.0.11
 
 stages:
-  - template: stage-with-burgr-notifications.yml@commonTemplates
```

Comment:
Everything was dropped from [this template](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_git/pipelines-yaml-templates?path=/stage-with-burgr-notifications.yml).